db/model/app.model.js

```diff
@@ -21,4 +21,17 @@ exports.selectArticleByArticleId = (article_id) => {
          }
          return rows
     })
+}
+
+//Task 5 CORE: GET /api/articles
+exports.selectAllArticles = () => {
+    const selectSQLStr = "SELECT a.author,a.title,a.article_id, a.topic,a.created_at,a.votes,a.article_img_url, " +
+                         "(SELECT count(b.article_id)::int FROM comments b WHERE b.article_id = a.article_id) AS comment_count " + 
+                         "FROM articles a "+
+                         "ORDER BY a.created_at DESC ;";
```

Comment:
This is very interesting, is there a reason you went for a sub query instead of using a join? Is there also a reason you are using the letter b to reference the comments table name? This name seems a little arbitrary.

Just as an added note using backticks instead of quotation marks will allow you to make this whole string multi line instead of having to concatenate with + which makes it a little less readable.